src/ucs/datastruct/ptr_map.inl

```diff
@@ -0,0 +1,93 @@
+/**
+* Copyright (C) Mellanox Technologies Ltd. 2020.  ALL RIGHTS RESERVED.
+*
+* See file LICENSE for terms.
+*/
+
+#ifndef UCS_PTR_MAP_INL_
+#define UCS_PTR_MAP_INL_
+
+#include "ptr_map.h"
+
+BEGIN_C_DECLS
+
+KHASH_IMPL(ucs_ptr_map_impl, ucs_ptr_map_key_t, void*, 1,
+           kh_int64_hash_func, kh_int64_hash_equal);
+
+static inline ucs_status_t ucs_ptr_map_init(ucs_ptr_map_t *map)
```

Comment:
maybe it should be void?

Reply:
from API perspective this is bad idea, also this is not data path